freedomofpress/securedrop
# Resolve Issue 4454

## Status

Ready for review 

## Description of Changes

Fixes #4454 .

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?
- Spin up local environment
- Confirm that "learn how to set it safest" warning that comes up as described in the issue is on the right, not left side of the screen.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.
As mentioned in the Issue thread, this does not try to tell the difference between Tor 8.5 and before.
## Checklist

### If you made changes to the server application code:


### If you made changes to `securedrop-admin`:


### If you made changes to the system configuration:


### If you made non-trivial code changes:


### If you made changes to documentation:
